src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -217,4 +227,21 @@ export class TaskListWrapperComponent implements OnInit {
       handleFatalErrors(error.status, this.router, WILDCARD_SERVICE_DOWN);
     });
   }
+
+  private loadTasksVersion1(): void {
+    if (this.wasBadRequest) {
+      this.refreshTasks();
+    } else {
+      this.doLoad();
+    }
+  }
+
+  private loadTasksVersion2(): void {
```

Comment:
This is identical to the one above. Why does it need two? Also, what's up with the naming convention?

Reply:
If @ronaldmansveld  has asked to do it this way I am ok. I dnt have a view on it.